api/codegen.sh

```diff
@@ -6,7 +6,7 @@ npx @rtk-query/codegen-openapi ./api/config/rhsm.ts &
 npx @rtk-query/codegen-openapi ./api/config/contentSources.ts &
 npx @rtk-query/codegen-openapi ./api/config/provisioning.ts &
 npx @rtk-query/codegen-openapi ./api/config/edge.ts &
-
+npx @rtk-query/codegen-openapi ./api/config/cloudapi.ts &
```

Comment:
Let's keep the newline in this case

```suggestion
npx @rtk-query/codegen-openapi ./api/config/cloudapi.ts &

```